### PR TITLE
Preview Panel strings

### PR DESCRIPTION
**Pull request by KamilGrzegorski-TomTom**
## Description

Hello all, please take a look at the following string proposal 

## Link to specs
link 

## Links to String Table
link 
## Attachments

photo 1 2 3

### CODE REVIEW

File: NK2UI/en-US/strings.yaml [hunk 1]
```diff
@@ -6,3 +6,23 @@ example1_string_id:
   needsTranslation: true
   translationTicket: none
   tableId: XX_XXXX_XXXX
+
+# Preview panel
+
+nk2ui_location_preview_drive_button:
```
Comment (JakubLipowski-TomTom) [inline NK2UI/en-US/strings.yaml:12]: Jakiś komentarz

File: NK2UI/en-US/strings.yaml [hunk 2]
```diff
@@ -6,3 +6,23 @@ example1_string_id:
...
+  text: "Drive"
+  description: "Button to start the navigation towards this location"
+  needsTranslation: true
+  translationTicket: none
+
+nk2ui_location_preview_location_decline_fallback_lable :
```
Comment (JakubLipowski-TomTom) [inline NK2UI/en-US/strings.yaml:18]: Jakiś drugi komentarz

File: NK2UI/en-US/strings.yaml (hunk 1 above)
Comment (KamilGrzegorski-TomTom) [reply NK2UI/en-US/strings.yaml:12]: I made changes to the code

Comment (JakubLipowski-TomTom) [reply NK2UI/en-US/strings.yaml:12]: Ok, it's ok now

**Review by JakubLipowski-TomTom — APPROVED** [review]

**Pull request merged**